hrjobcontract/js/test/controllers/modal/modal-change-reason_test.js

```diff
@@ -0,0 +1,116 @@
+define([
+    'common/moment',
+    'common/angularMocks',
+    'job-contract/app'
```

Comment:
We need `app.js` as we need `ContractRevisionService` for effective_date validation.